cmd/agent/gui/views/public/js/yaml.js

```diff
@@ -0,0 +1,118 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
```

Comment:
we need to add this library to the file `LICENSE-3rdparty.csv`

Reply:
Done :+1: